src/parsers/dr_ste_v1_tag_parser.py

```diff
@@ -281,13 +334,23 @@ def mlx5_ifc_ste_src_gvmi_qp_v1_bits_tag_parser(bin_str):
     }
 
 
-def mlx5_ste_v1_tag_parser(entry_format, tag, raw):
+switch_definer_parser = {
+    0x22: mlx5_ifc_ste_def22_v1_bits_tag_parser,
+    0x24: mlx5_ifc_ste_def24_v1_bits_tag_parser, 
+}
+
+def mlx5_ste_v1_tag_parser(entry_format, definer_id, tag, raw):
+        func, inner = None, None
 
-	if entry_format not in switch_tag_parser.keys():
-		print("Err: Unsupported STEv1 tag format")
-		return {}
+	if entry_format in switch_tag_parser.keys():
+	    func, inner = switch_tag_parser[entry_format]
+        elif definer_id not in [None, -1, "-1"] and int(definer_id, 16) in switch_definer_parser.keys():
+            definer_id = int(definer_id, 16)
```

Comment:
don't override the func param, just do this on the next line

Reply:
done